apps/src/utils.js

```diff
@@ -235,7 +235,13 @@ export function isInfiniteRecursionError(err) {
 export function browserSupportsCssMedia() {
   var styleSheets = document.styleSheets;
   for (var i = 0; i < styleSheets.length; i++) {
-    var rules = styleSheets[i].cssRules || styleSheets[i].rules;
+    var rules;
+    try {
+      rules = styleSheets[i].cssRules || styleSheets[i].rules;
+    } catch (e) {
+      // Some versions of Chrome give an error when accessing cssRules.
+      return false;
```

Comment:
It seems like any exception in this case indicates that the browser does not support CSS Media, so returning false seems correct.

Reply:
I guess what I'm asking is: what is the specific error text the browser throws in the case we're trying to catch?  Do we want to re-throw if we get a different error text?  (Indicating an error we didn't expect, and maybe don't want to blanket-catch-all.)

Reply:
Hey Josh, good question. We kind of don't care what the error is here. we touch a bunch of stylesheets here only to detect if it will be safe to touch the same stylesheets in the same way later.

in this particular case, the logic we are protecting has already been decommissioned (in https://github.com/code-dot-org/code-dot-org/pull/14312), so the solution, shown in Brad's PR link below, is to remove this detection logic entirely.